app/src/main/java/com/wafflestudio/snutt2/views/RootActivity.kt

```diff
@@ -438,8 +438,13 @@ class RootActivity : AppCompatActivity() {
         composableAnimated<NavigationDestination.ThemeModeSelect> { ColorModeSelectPage() }
         if (BuildConfig.DEBUG) {
             composableAnimated<NavigationDestination.LectureDiary> { DiaryListPage() }
-            composableAnimated<NavigationDestination.LectureDiaryWrite> {
+            composableAnimated<NavigationDestination.LectureDiaryWrite> { entry ->
+                val lectureId = entry.arguments?.getString("lectureId") ?: "686e8d3c2afaf11b888e2722" // TODO: NULL일때 에러 던지기
+                val lectureName = entry.arguments?.getString("lectureName") ?: "알고리즘"
+                val lectureNumber = entry.arguments?.getString("lectureNumber") ?: "53131"
```

Comment:
```suggestion
                val lectureId = entry.toRoute<NavigationDestination.LectureDiaryWrite>().lectureId ?: return@composableAnimated
                val lectureName = entry.toRoute<NavigationDestination.LectureDiaryWrite>().lectureName ?: return@composableAnimated
                val lectureNumber = entry.toRoute<NavigationDestination.LectureDiaryWrite>().lectureNumber ?: return@composableAnimated
```
toRoute 확장함수 사용하기

값 없을 때 그냥 return 하면 흰화면 나오긴 하는데, return 하지 말고 홈화면 보내버려도 될듯